ACE/ACE-INSTALL.html

```diff
@@ -2507,7 +2576,7 @@ <h1><a name="installnotes">Installation Notes</a></h1>
       When building without network support you will need the ace_for_tao
       subsetting functionality enabled. For this add <code>ace_for_tao = 1</code>
       to your <code>bin/MakeProjectCreator/config</code> file and regenerate the
-      GNU makefiles as described <a href="#power">here</a>.</p><p>
+      GNU makefiles as described <a href="#buildfiles">here</a>.</p><p>
```

Comment:
Where is the buildfiles section?

Reply:
```suggestion
      GNU makefiles as described <a href="#power">here</a>.</p><p>
```

Reply:
I renamed it but decided to revert it because it could break a link. I just missed this one.